.circleci/config.yml

```diff
@@ -451,33 +455,41 @@ jobs:
           <<: *defaults_build_docker_publish
       - run:
           <<: *defaults_slack_announcement
-        
-  # deploy:
-  #   executor: helm-kube
-  #   steps:
-  #     - checkout
-  #     - run:
-  #         name: Install AWS CLI dependencies
-  #         command: *defaults_awsCliDependencies
-  #     - run:
-  #         name: setup environment vars for release/snapshot
-  #         command: ./.circleci/_set_up_deploy_envs.sh
-  #     - run:
-  #         <<: *defaults_deploy_prequisites
-  #     - run:
-  #         <<: *defaults_deploy_config_kubernetes_cluster
-  #     - run:
-  #         <<: *defaults_deploy_config_kubernetes_credentials
-  #     - run:
-  #         <<: *defaults_deploy_config_kubernetes_context
-  #     - run:
-  #         <<: *defaults_deploy_set_kubernetes_context
-  #     - run:
-  #         <<: *defaults_deploy_configure_helm
-  #     - run:
-  #         <<: *defaults_deploy_install_or_upgrade_helm_chart
-
- 
+
+  publish-init-central-ledger:
+    executor: default-machine
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp
+      - run:
+          name: Load the pre-built docker image for init-central-ledger from workspace
+          command: docker load -i /tmp/docker-image-init-central-ledger.tar
+      - run:
+          <<: *defaults_build_docker_login
+      - run:
+          name: Setup environment vars for release/snapshot
+          command: ./.circleci/_set_up_deploy_envs.sh
+      - run:
+          name: Re-tag pre built image
+          command: |
+            docker tag $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:$CIRCLE_TAG $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME-init-central-ledger:$RELEASE_TAG
+      - run:
+          name: Publish Docker image $CIRCLE_TAG & Latest tag to Docker Hub for init-central-ledger
+          command: |
+            echo "Publishing $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME-init-central-ledger:$CIRCLE_TAG"
+            docker push $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME-init-central-ledger:$CIRCLE_TAG
+            echo "Publishing $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME-init-central-ledger:$RELEASE_TAG"
+            docker push $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME-init-central-ledger:$RELEASE_TAG
+      - run:
+          name: Slack announcement for tag releases
+          command: |
+            curl -X POST \
+              $SLACK_WEBHOOK_ANNOUNCEMENT \
+              -H 'Content-type: application/json' \
+              -H 'cache-control: no-cache' \
+              -d "{\"text\": \"*${CIRCLE_PROJECT_REPONAME}-init-central-ledger* - Release \`${CIRCLE_TAG}\`: https://github.com/mojaloop/${CIRCLE_PROJECT_REPONAME}/releases/tag/${CIRCLE_TAG}\"}"
```

Comment:
Do we want a separate Slack announcement for this? (Maybe? Maybe not?)

Reply:
It's a released artifact. I'd say so. Though, I'm not sure where these announcements are made.

Reply:
In the ML slack.